### DISCUSSION TITLE
5.0 rc 版本延期（10月 -> 11月）

### DISCUSSION THREAD

**Discussion by awmleer** [ant-design/ant-design-mobile#4247] [Announcements]
在之前的 5.0 roadmap #3924 中，我们原计划在 10 月份开始推送 rc 版本，但是由于现在的 API 还不足够稳定，我们最近的几次发布还在不断的产生 break change，所以 rc 版本预计将延期到 11 月底之前开始推送，5.0 正式版本的发布时间待定。预计在最近 2 周内我们会对 5.0 roadmap 进行一些具体的调整和更新。

当然，rc 版本的推迟并不意味着组件库的研发进度有所异常，相信大家也注意到了我们目前正处在全力开发中的状态，组件库的特性需求和 bug 修复都在比较及时地进行处理。只是我们希望确保 5.0 的正式发布是足够完善和成熟的，能够达到我们心中一个优质组件库的标准。 希望大家谅解～

**Comment by xiaohuoni** [comment]
监工身份认领👀

**Comment by zqran** [comment]
期待 🤩

**Comment by Me-UserName** [comment]
加油，期待
